packages/documentation/src/pages/platform/unit-testing.mdx

```diff
@@ -0,0 +1,203 @@
+## How do unit tests work?
+
+Table of Contents
+ - [Unit](#unit)
+ - [End to End](#end-to-end)
```

Comment:
I think we should make this doc entirely focused on how our unit tests work. There's a separate spot in the info architecture for how end to end tests work, so that would be a separate doc (with I think a sub section for the a11y tests).